tests/python/openassetio/hostApi/test_manager.py

```diff
@@ -761,3 +787,32 @@ def is_defined_in_python(method):
     # purpose. getsource and similar raise a TypeError if the supplied
     # object isn't a function, so the flow control is simpler this way.
     return inspect.isfunction(method)
+
+
+def is_implemented_once(cls, method_name):
+    """
+    Check if a method is only implemented once in the class hierarchy.
+
+    In particular, this is useful to ensure we aren't hiding a (C++)
+    base class method with a (Python) subclass method. I.e. this checks
+    that we haven't migrated a method to C++ then forgot to remove the
+    Python implementation.
```

Comment:
[fix] Can we add a note that it is a backup check incase we forget to update the `is_implemented_in_python` based tests?

Whilst were there, could we add a `@todo` to remember to remove all these tests once the `Manager.py` is deleted?

Reply:
Added notes in cb6b3684, but then migrated notes  to conftest.py in bbf88560